crates/sncast/tests/e2e/account/create.rs

```diff
@@ -185,12 +186,15 @@ pub async fn test_happy_case_add_profile() {
         "my_account",
     ];
 
-    let snapbox = runner(&args).current_dir(tempdir.path());
-    let output = snapbox.assert().success();
+    let output = runner(&args).current_dir(tempdir.path()).assert();
+    let manifest_path = Utf8PathBuf::from_path_buf(tempdir.path().join("snfoundry.toml"))
+        .expect("creating manifest path should be possible")
+        .canonicalize_utf8()
+        .expect("canonicalize manifest path should be possible");
```

Comment:
```suggestion
    let config_path = Utf8PathBuf::from_path_buf(tempdir.path().join("snfoundry.toml"))
        .unwrap()
        .canonicalize_utf8()
        .unwrap();
```
For consistency and to reserve the term manifest specifically for the `Scarb.toml` file in this context. The same suggestion for other modified tests.